bpfd/src/multiprog/xdp.rs

```diff
@@ -69,13 +66,23 @@ impl XdpDispatcher {
         };
 
         debug!("xdp dispatcher config: {:?}", config);
-
+        let image = BytecodeImage::new(
+            "quay.io/bpfd/xdp-dispatcher:v1".to_string(),
```

Comment:
Should we be doing the same for the TC dispatcher as well?

Reply:
I was going to ask the same question :-).  It seems like it would be a good idea.